log.go

```diff
@@ -392,6 +392,7 @@ func (l *Logger) Log() *Event {
 // Arguments are handled in the manner of fmt.Print.
 func (l *Logger) Print(v ...interface{}) {
 	if e := l.Debug(); e.Enabled() {
+		e.CallerSkipFrame(1)
```

Comment:
Please inline it the line below.

Reply:
Done. 3326547548fb5fe2a8e6d4db82a7f692a17d6073